modules/openapi-generator/src/main/resources/typescript/services/ObservableAPI.mustache

```diff
@@ -14,10 +17,10 @@ import { {{{ classname }}} } from "{{{ importPath }}}";
 
 export abstract class AbstractObservable{{classname}} {
     {{#operation}}
-    public abstract {{nickname}}({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}options?: Configuration): Observable<{{{returnType}}}{{^returnType}}void{{/returnType}}>;
+    public abstract {{nickname}}({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}options?: Configuration{{^useInversify}} | Middleware[]{{/useInversify}}): Observable<{{{returnType}}}{{^returnType}}void{{/returnType}}>;
```

Comment:
is this api change backwards compatible?

Reply:
removed to opt for adding merging to optional version of current signature instead